lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/generic/JobDriver.java

```diff
@@ -590,7 +607,8 @@ public void onNext(final StartTime startTime) {
           LOG.log(Level.INFO, "Finished CLR bridge setup for {0}", startTime);
         }
 
-        NativeInterop.callClrSystemOnStartHandler();
+        bridge.callClrSystemOnStartHandler();
```

Comment:
After reading your comments yesterday, I think this is a good idea. I can remove replacing the call to callClrSystemOnStartHandler() so the new code is not operative by default and then set up the new code to operate via a flag (I'll need to give this design some thought). Then I can gather some performance numbers.